show how to always use the git blame ignore file (#18198)
github/docs: content/repositories/working-with-files/using-files/viewing-a-file.md

git blame --ignore-revs-file .git-blame-ignore-revs
```

> Or to configure your local git so it always ignores the revs in that file:

Reviewer: ```suggestion
You can also configure your local git so it always ignores the revs in that file:
```
